src/cpu/aarch64/brgemm/jit_brgemm_kernel.cpp

```diff
@@ -1254,14 +1270,21 @@ void jit_brgemm_kernel_t::set_A_B_matrices() {
     add(reg_aux_B, reg_aux_B, reg_b_offset);
 }
 
-void jit_brgemm_kernel_t::dot_product(ZReg v1, ZReg v2, ZReg v3) {
+void jit_brgemm_kernel_t::dot_product(ZReg v1, ZReg v_b, ZReg v_a) {
```

Comment:
`v1`,`v2` and `v3` is unclear, although it is consistent. `v1`,`v_b` and `v_a` feels like the right direction, but is more confusing.
- Why is `b` first?
- What is `v1` in this case, if `a` and `b` reference the A and B matrices, then it probably makes sense to give `v1` a name with a similar theme, maybe `v_acc`?

Reply:
> One alternative is to switch the order in the function call so source comes first and weights second, and update the code inside.

I'll leave this to you to decide which way makes more sense